src/transformers/__init__.py

```diff
@@ -401,6 +400,11 @@
         "TrOCRConfig",
         "TrOCRProcessor",
     ],
+    "models.tvlt": [
+        "TVLT_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "TvltConfig",
+        "TvltAudioFeatureExtractor",
+    ],
```

Comment:
```suggestion
    "models.tvlt": [
        "TVLT_PRETRAINED_CONFIG_ARCHIVE_MAP",
        "TvltConfig",
        "TvltAudioFeatureExtractor",
        "TvltProcessor",
    ],
```
See also my comment below. Also does the feature extractor have any dependency?